src/Raven.Studio/wwwroot/App/views/database/settings/databaseSettings.html

```diff
@@ -0,0 +1,231 @@
+        <div class="btn-group">
+            <button class="btn btn-default dropdown-toggle filter-entries" data-toggle="dropdown">
+                <span data-bind="visible: groupToShow() === 'allEntries'">All Entries</span>
+                <span data-bind="visible: groupToShow() === 'databaseEntries'">Database Entries</span>
+                <span data-bind="visible: groupToShow() === 'customizedDatabaseEntries'">Customized Database Entries</span>
+                <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu">
+                <li title="Show all entries per database & server-wide"><a href="#" data-bind="click: _.partial(setGroupToShow, 'allEntries')">All Entries</a></li>
+                <li title="Show only entries per this database"><a href="#" data-bind="click: _.partial(setGroupToShow, 'databaseEntries')">Database Entries</a></li>
+                <li title="Show entries that are customized per this database"><a href="#" data-bind="click: _.partial(setGroupToShow, 'customizedDatabaseEntries')">Customized Database Entries</a></li>
+            </ul>
+        </div>
+        <div class="filter-entries">
+            <input class="form-control" placeholder="Filter Keys" data-bind="textInput: filterKeys" />
+        </div>
+        <div>
+            <button class="btn btn-default" data-bind="click: refresh, css: { 'btn-spinner': spinners.refresh }, disable: spinners.refresh() || spinners.save()" title="Reload settings">
+                <i class="icon-refresh"></i><span>Refresh</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-noshrink summary-buttons">
+        <div class="flex-separator"></div>
+        <div data-bind="visible: viewMode() === 'summaryMode'">
+            <button class="btn btn-sm btn-default" data-bind="click: switchToEditMode" title="Edit the Database Settings">
+                <i class="icon-edit"></i><span>Edit Entries</span>
+            </button>
+        </div>
+        <div data-bind="visible: viewMode() === 'editMode', enable: !isSaveEnabled()">
+            <button class="btn btn-sm btn-default" data-bind="click: exitEditMode" title="View all effective values in a condensed view">
+                <i class="icon-table"></i><span>View Summary</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-stretch-items absolute-fill content-margin entries-grid" data-bind="visible: viewMode() === 'editMode'">
+        <div class="flex-vertical">
+            <div class="categories-list-container flex-vertical flex-grow" style="position:relative;">
+                <virtual-grid class="resizable virtual-grid" params="controller: categoriesGridController"></virtual-grid>
+            </div>
+        </div>
+        <div class="flex-grow slidein-style">
+            <div data-bind="template: { name: 'db-settings-no-matching-entries-template' }, visible: !isAnyMatchingEntries()"></div>
```

Comment:
let's inline this template hrere.

Reply:
Actually, I prefer that we leave this template since I would like to use it in **both** grids.